### PR TITLE
Bump kubernetes 1.26.10 in release-1.15 

### PR DESCRIPTION
**Pull request by Shelley-BaoYue**
**What type of PR is this?**

/kind bug
/kind cleanup

**What this PR does / why we need it**:

1. bump kubernetes dependency version to newest patch version 1.26.10, for Kubernetes community has fixed some problem since 1.26.7.  FYI:  https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.26.md 
2. fix serviceaccounttoken cannot be deleted in edge DB. More details see in PR #5133

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -292,40 +292,40 @@ replace (
...
-	k8s.io/controller-manager => github.com/kubeedge/kubernetes/staging/src/k8s.io/controller-manager v1.26.7-kubeedge1
-	k8s.io/cri-api => github.com/kubeedge/kubernetes/staging/src/k8s.io/cri-api v1.26.7-kubeedge1
-	k8s.io/csi-api => github.com/kubeedge/kubernetes/staging/src/k8s.io/csi-api v1.26.7-kubeedge1
-	k8s.io/csi-translation-lib => github.com/kubeedge/kubernetes/staging/src/k8s.io/csi-translation-lib v1.26.7-kubeedge1
-	k8s.io/dynamic-resource-allocation => github.com/kubeedge/kubernetes/staging/src/k8s.io/dynamic-resource-allocation v1.26.7-kubeedge1
-	k8s.io/gengo v0.0.0 => k8s.io/gengo v0.26.7
+	k8s.io/api => github.com/Shelley-BaoYue/kubernetes/staging/src/k8s.io/api v1.26.10-kubeedge1-by
+	k8s.io/apiextensions-apiserver => github.com/Shelley-BaoYue/kubernetes/staging/src/k8s.io/apiextensions-apiserver v1.26.10-kubeedge1-by
+	k8s.io/apimachinery => github.com/Shelley-BaoYue/kubernetes/staging/src/k8s.io/apimachinery v1.26.10-kubeedge1-by
+	k8s.io/apiserver => github.com/Shelley-BaoYue/kubernetes/staging/src/k8s.io/apiserver v1.26.10-kubeedge1-by
```
Comment (fisherxu) [inline go.mod:298]: Update to `kubeedge/kuberrnetes` if it's ready.

Comment (Shelley-BaoYue) [reply go.mod:298]: > Update to `kubeedge/kuberrnetes` if it's ready.

If there are no objection about this update and PR #5133,  I'm about to push a new official tag of kubeedge/kubernetes v1.26.10.

File: edge/pkg/edged/kubeclientbridge/typed/core/v1/serviceaccount_bridge.go [hunk 2]
```diff
@@ -46,6 +47,11 @@ func (c *ServiceAccountsBridge) CreateToken(ctx context.Context, serviceAccountN
 	return c.MetaClient.ServiceAccountToken().GetServiceAccountToken(c.ns, serviceAccountName, tokenRequest)
 }
 
+func (c *ServiceAccountsBridge) Delete(ctx context.Context, podUID string, opts metav1.DeleteOptions) error {
+	c.MetaClient.ServiceAccountToken().DeleteServiceAccountToken(types.UID(podUID))
+	return nil
```
Comment (fisherxu) [inline edge/pkg/edged/kubeclientbridge/typed/core/v1/serviceaccount_bridge.go:52]: `Delete` will delete the `ServiceAccount` in the database when pod is deleted. @WillardHu PTAL
/approve

**Pull request merged**